_posts/2022-04-26-omero-5-6-4.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.4
+---
+
+Today we are releasing OMERO.server 5.6.4 which uses OMERO.blitz 5.5.10
+and includes Bio-Formats 6.9.1, see [release announcement]({{ site.baseurl }}/2022/04/20/bio-formats-6-9-1.html).
+The Bio-Formats Memoizer cache **will be invalidated** on upgrade from OMERO.server 5.6.3.
```

Comment:
Also worth a quick mention of the OMERO.scripts update as several fixes have been made (do we have a CHANGELOG?)

Reply:
unfortunately no changelog!!